master/internal/db/postgres_experiments.go

```diff
@@ -922,52 +922,6 @@ func (db *PgDB) DeleteExperiments(ctx context.Context, ids []int) error {
 		}
```

Comment:
Gr, Github won't let me comment on the correct line, but since this operation is now only one query (`DELETE FROM trials...`) rather than a sequence of queries that need to happen all in one go, we no longer need to use a transaction. And since transactions can be costly from a performance standpoint, I recommend getting rid of

```go
tx, err := Bun().BeginTx(ctx, nil)
defer func(){
    txErr := tx.Rollback()
    // ...etc.
}()
// ...etc.
tx.NewDelete() // ...etc.
tx.Commit()
```
and instead just doing
```go
Bun().NewDelete() // ...etc.
```
Another nice benefit of cutting the transaction is that it's less code.